src/snipe/api/multisig_reference_QC.py

```
@@ -0,0 +1,901 @@
+import warnings
+from typing import Any, Dict, Iterator, List, Optional, Union
+
+import numpy as np
+from scipy.optimize import OptimizeWarning, curve_fit
+from snipe.api.snipe_sig import SnipeSig
+from snipe.api.enums import SigType
+import os
+import requests
+from tqdm import tqdm
+from urllib.parse import urlparse
+from typing import Optional
```

Ruff

Redefinition of unused `Optional` from line 4

Remove definition: `Optional`

(F811)